testutils/models.go

```diff
@@ -51,6 +51,33 @@ func InitJetDropDB(pulse models.Pulse) models.JetDrop {
 	}
 }
 
+// GenerateJetDropsWithSomeJetID returns a list of JetDrops with some JetID and ascending pulseNumber
+func GenerateJetDropsWithSomeJetID(t *testing.T, jCount int) (string, []models.JetDrop, []models.Pulse) {
```

Comment:
This func generate one jetDrop for pulse. Its better to generate several jetDrops in every pulse, that way tests will check if different params work for that case (several jetDrops in one pulse).